components/rmrk/Gallery/Gallery.vue

```diff
@@ -160,6 +160,11 @@ export default class Gallery extends mixins(
   private hasPassionFeed = false
   private passionList: string[] = []
 
+  // hiding until passion feet gets implemented for bsx
+  get isBsx() {
+    return this.urlPrefix === 'bsx'
+  }
```

Comment:
I would hide passion feed in general imo

Reply:
@yangwao do we want to hide passion feed in general?

Reply:
yes, seems it's poorly tested and do more harm than good :)
- drops squidy often (query is huge)
- bsx doesn't work
- name other I didn't see yet

Reply:
ok, passion feed hidden.